src/main/kotlin/Handler.kt

```diff
@@ -22,30 +24,60 @@ class Handler : RequestHandler<Request, ByteArray?> {
     val logger = Logger.getLogger("Handler")
     val keyGeneration = KeyGeneration()
 
-    override fun handleRequest(input: Request, context: Context?): ByteArray? {
+    override fun handleRequest(input: Request, context: Context): ByteArray? {
```

Comment:
I think this will break existing tests, I would just pass in another field in the request called something like "isDelete" which **if present** and **if true** does a delete, not a retrieval

Reply:
I have changed it accoridngly . Thanks Steve